tests/filesystem/test_filesystem.py

```diff
@@ -2,7 +2,20 @@
 # -*- coding: utf-8 -*-
 # vim: ai ts=4 sts=4 et sw=4 nu
 
-from zimscraperlib.filesystem import get_file_mimetype, get_content_mimetype
+from zimscraperlib.filesystem import (
+    get_file_mimetype,
+    get_content_mimetype,
+    MIME_OVERRIDES,
+)
+
+
+def test_mime_overrides():
+    assert len(MIME_OVERRIDES) == 1
+    verified_overrides = {
+        "image/svg": "image/svg+xml",
+    }
+    for key, val in verified_overrides.items():
+        assert MIME_OVERRIDES[key] == val
```

Comment:
That's what what we want to test. We want to test that getting the mime of an SCG file returns our override (either because the libmagic sends it or because it was overriden (that we don't care)